contracts/CEtherDelegateTempExploitAccounting.sol

```diff
@@ -0,0 +1,221 @@
+contract CEtherDelegateTempExploitAccounting is CDelegateInterface, CTokenStorage, CErc20Storage, Exponential {
+    /**
+     * @notice Construct an empty delegate
+     */
+    constructor() public {}
+
+    /**
+     * @notice Gets balance of this contract in terms of Ether, before this message
+     * @dev This excludes the value of the current message, if any
+     * @return The quantity of Ether owned by this contract
+     */
+    function getCashPrior() internal view returns (uint) {
+        (MathError err, uint startingBalance) = subUInt(address(this).balance, msg.value);
+        require(err == MathError.NO_ERROR);
+        return startingBalance;
+    }
+
+    /**
+     * @notice Calculates the exchange rate from the underlying to the CToken
+     * @dev This function does not accrue interest before calculating the exchange rate
+     * @return Calculated exchange rate scaled by 1e18
+     */
+    function exchangeRateStored() public view returns (uint) {
+        (MathError err, uint result) = exchangeRateStoredInternal();
+        require(err == MathError.NO_ERROR, "exchangeRateStored: exchangeRateStoredInternal failed");
+        return result;
+    }
+
+    /**
+     * @notice Calculates the exchange rate from the underlying to the CToken
+     * @dev This function does not accrue interest before calculating the exchange rate
+     * @return (error code, calculated exchange rate scaled by 1e18)
+     */
+    function exchangeRateStoredInternal() internal view returns (MathError, uint) {
+        uint _totalSupply = totalSupply;
+        if (_totalSupply == 0) {
+            /*
+             * If there are no tokens minted:
+             *  exchangeRate = initialExchangeRate
+             */
+            return (MathError.NO_ERROR, initialExchangeRateMantissa);
+        } else {
+            /*
+             * Otherwise:
+             *  exchangeRate = (totalCash + totalBorrows - (totalReserves + totalFuseFees + totalAdminFees)) / totalSupply
+             */
+            uint totalCash = getCashPrior();
+            uint cashPlusBorrowsMinusReserves;
+            Exp memory exchangeRate;
+            MathError mathErr;
+
+            (mathErr, cashPlusBorrowsMinusReserves) = addThenSubUInt(totalCash, totalBorrows, add_(totalReserves, add_(totalAdminFees, totalFuseFees)));
+            if (mathErr != MathError.NO_ERROR) {
+                return (mathErr, 0);
+            }
+
+            (mathErr, exchangeRate) = getExp(cashPlusBorrowsMinusReserves, _totalSupply);
+            if (mathErr != MathError.NO_ERROR) {
+                return (mathErr, 0);
+            }
+
+            return (MathError.NO_ERROR, exchangeRate.mantissa);
+        }
+    }
+
+    /**
+     * @notice Called by the delegator on a delegate to initialize it for duty
+     * @param data The encoded bytes data for any initialization
+     */
+    function _becomeImplementation(bytes calldata data) external {
+        // Shh -- currently unused
+        data;
```

Comment:
https://github.com/Rari-Capital/compound-protocol/pull/10/files

Reply:
Let's remove this line then?